mapc_rhbp_ettlinger/TaskAssignment      assignment
uint32                                      id